Source/Core/InputCommon/ControlReference/ExpressionParser.cpp

```diff
@@ -539,13 +546,13 @@ class HotkeyExpression : public Expression
 
     // We must update our suppression with valid pointers.
     if (m_suppressor)
-      EnableSuppression();
+      EnableSuppression(true);
   }
 
 private:
-  void EnableSuppression() const
+  void EnableSuppression(bool force = false) const
   {
-    if (!m_suppressor)
+    if (!m_suppressor || force)
```

Comment:
I'm confused why this `force` is needed.  The only place you made it not the default is where the `m_suppressor` is already true?

Reply:
`EnableSuppression` is called in two places and I didn't want to change the other call to it, so to keep changes to a min I added a `force` param which defaults to false.
Before it was calling `EnableSuppression` only if `m_suppressor` was true, but then the function inside would only proceed if it was false, which made no sense.